### DISCUSSION TITLE
How to fix cant run Cosmos os with a build error

### DISCUSSION THREAD

**Discussion by IDONTCAREE44** [CosmosOS/Cosmos#2570] [Q&A] [answered]
Hi, i did everything to set up cosmos userkit. please I really need Help i restarted the installiton of everything i installed to setup to make os with c#.. i download Visual studio commonuinty 2022, Cosmos userkit, .NET core 2.1 , .net microsoft framework 4.6.2,  and Of course VMware workstation player 17 and it doesnt show anything when i open cosmos c# project in the output or error list!, i cant run because it give me this error that i have no idea about.
![error](https://user-images.githubusercontent.com/123968781/215518441-622f23da-5089-4c3b-8c38-6a6663825493.png)

then i changed mind and i thought that's because I need to create new Virtual machine on VMware and I did. after that I realized I need Iso file For my operating system cosmos os.. i got to the visual studio project folder. then bin>debug and then net 6.0, its not supposed to be net 6.0 i know.. after that i got inside that folder i didn't find any thing... all i found is publish folder only and i have no idea where is the iso. i kept making new projects and doing alot of attempts. i did 67 attempts BRO AND I DIDN'T FIND ANYTHING! IT TOOK ME 4 DAYS! Please help needed and i appreciate that..

**Comment by ascpixi** [comment, marked as answer]
The project build has failed due to a missing library that `yasm` requires, the assembler utilized by Cosmos. You have to install [Microsoft Visual C++ 2010 Redistributable](https://www.microsoft.com/en-us/download/details.aspx?id=26999). Make sure to download it for x64.

For future reference: `-1073741515` is hexadecimal `0xC0000135`, which translates to STATUS_DLL_NOT_FOUND in the [`ntstatus.h`](https://learn.microsoft.com/en-us/openspecs/windows_protocols/ms-erref/596a1078-e883-4972-9bbc-49e60bebca55) Windows header.

**Comment by IDONTCAREE44** [comment]
thanks for the attention! is that all?

**Reply by ascpixi** [reply]
Yep, that should be it!

**Reply by IDONTCAREE44** [reply]
I downloaded the setup and installed the app. I **created** new cosmos c# project but its still the same story as before. oh and plus the icon of my cosmos os in the solution explorer doesn't show up. it just show up a smile face

**Reply by IDONTCAREE44** [reply]
oh wait nevermind its working now. thank you very much

**Reply by ascpixi** [reply]
> oh wait nevermind its working now. thank you very much

No problem! If you don't mind, please mark my answer as accepted, thanks! :)

**Reply by IDONTCAREE44** [reply]
wait how im new to github today lol

**Reply by ascpixi** [reply]
![image](https://user-images.githubusercontent.com/44982772/215582160-0ff09319-b160-4602-8020-9c53a7081778.png)
You should have a button like this.

**Reply by IDONTCAREE44** [reply]
nope idk why
